tsconfig.json

```diff
@@ -6,8 +6,12 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "jsx": "react",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "jsx": "react-jsx",
```

Comment:
Unsure how this got changed. This is required so that `React` is imported correctly.